personal_profile/tests.py

```diff
@@ -28,16 +33,41 @@ def profile_1(db, user_1):
                                                                content_type='image/jpeg'),
                                 resume=SimpleUploadedFile('test_resume.txt',
                                                           b'these are the contents of the txt file'))
+    profile_1.save()
     return profile_1
 
 
+@pytest.mark.django_db
 class TestProfileModel:
-    # Testing different Personal Profile attributes
-
-    def test_newprofile(self, profile_1, user_1):
-        assert profile_1.company == 'Test Company'
-        assert profile_1.user == user_1
-        assert profile_1.about == 'Test About'
-        assert profile_1.birth_date == datetime.date(1995, 12, 10)
-        assert profile_1.profile_pic is not None
-        assert profile_1.resume is not None
+    # Testing different Personal Profile saving and deletion
+
+    def test_profile_is_saved_correctly(self, profile_1, user_1):
```

Comment:
see comments in other PRs.
Filter the profile and compare its content against returned data fixture

Reply:
done